modules/nf-core/multiqc/tests/e2e/main.nf.test

```diff
@@ -0,0 +1,29 @@
+nextflow_pipeline {
+
+    name "Test MultiQC integration"
+    script "./main.nf"
+
+    tag "fastqc"
+    tag "bowtie2/align"
+
+    test("Single-Read") {
+
+        when {
+            params {
+                outdir   = "$outputDir"
+            }
+        }
+
+        then {
+            assertAll (
+            { assert process.success },
+            // NOTE The report contains the date inside it, which means that the md5sum is stable per day, but not longer than that. So you can't md5sum it.
```

Comment:
There's a MultiQC option to omit the date from the report, I think

Reply:
That was just copied from the stock test 🙃 

> I think
😆